content/zh/docs/ops/deployment/deployment-models/index.md

```diff
@@ -98,6 +98,16 @@ Kubernetes 在每个集群都默认配置此行为，这有助于限制由错误
 您可以根据[网络](#network-models)和云提供商所支持的选项来配置集群间通信。
 例如，若两个集群位于同一基础网络，则可以通过简单地配置防火墙规则来启用跨集群通信。
 
+<!--
```

Comment:
needs remove this?

Reply:
I think adding the content of the original English document through annotations can help the next person who changes the document to better track the changes to the English document, what do you think?

Reply:
I'm -1 for this, in zh docs, if we keep the english docs, it means that we should keep updating both en and zh docs if any changes made in en docs. For a comparison purpose, referring to the original en docs is the most direct/easy way:)

Reply:
shouldn't zh docs track changes to en docs? using the method of refer to the original en docs may cause the content to be uneven and confusing to readers. Does this lose the meaning of document localization?

Reply:
We should track the changes from en docs, but remaining the en doc in zh is not a good way.

Reply:
got it thx~, #10951 has done this update, I will close this PR.